packages/sdk-react/src/core/theme/mui-monite/index.ts

```diff
@@ -0,0 +1,1187 @@
+            color: statusBackgroundColors.draft,
+          },
+          '&.Monite-SummaryCard-StatusTypography-new': {
+            color: statusBackgroundColors.new,
+          },
+          '&.Monite-SummaryCard-StatusTypography-approve_in_progress': {
+            color: statusBackgroundColors.approve_in_progress,
+          },
+          '&.Monite-SummaryCard-StatusTypography-paid': {
+            color: statusBackgroundColors.paid,
+          },
+          '&.Monite-SummaryCard-StatusTypography-waiting_to_be_paid': {
+            color: statusBackgroundColors.waiting_to_be_paid,
+          },
+          '&.Monite-SummaryCard-StatusTypography-rejected': {
+            color: statusBackgroundColors.rejected,
+          },
+          '&.Monite-SummaryCard-StatusTypography-partially_paid': {
+            color: statusBackgroundColors.partially_paid,
+          },
+          '&.Monite-SummaryCard-StatusTypography-canceled': {
+            color: statusBackgroundColors.canceled,
+          },
+          '&.Monite-SummaryCard-StatusTypography-all': {
+            color: statusBackgroundColors.all,
+          },
+          '&.Monite-SummaryCard-AmountTypography': {
+            fontSize: 20,
+            marginTop: 4,
+          },
+        },
+      },
+    },
+    MuiFormLabel: {
+      styleOverrides: {
+        root: {
+          '&.Mui-disabled': {
+            color: 'black',
+          },
+        },
+      },
+    },
+    MuiInputLabel: {
+      defaultProps: {
+        shrink: true,
+      },
+      styleOverrides: {
+        root: {
+          fontSize: '14px',
+          transform: 'none',
+          position: 'static',
+          color: 'black',
+          minHeight: '28px',
+          '&.Mui-disabled': {
+            color: 'black',
+          },
+        },
+        asterisk: {
+          color: statusColors.red.color,
+          transform: 'scale(2) translate(6px, 2px)',
+          display: 'inline-block',
+          width: '20px',
+        },
+      },
+    },
+    MuiInputBase: {
+      styleOverrides: {
+        root: {
+          fontWeight: 400,
+          borderRadius: moniteTheme.borderRadius * 2.67,
+          minHeight: '40px',
+          '& .MuiInputBase-input': {
+            height: '40px',
+            lineHeight: '40px',
+            padding: '0 14px', // Adjust padding if needed
+            boxSizing: 'border-box',
+          },
+          '& .MuiInputBase-inputAdornedStart': {
+            display: 'flex',
+            alignItems: 'center',
+          },
+        },
+      },
+    },
+    MuiOutlinedInput: {
+      styleOverrides: {
+        // Editor borders
+        notchedOutline: {
+          borderColor: 'neutral.80',
+        },
+        root: {
+          '&.Mui-disabled .MuiOutlinedInput-notchedOutline, .MuiOutlinedInput-notchedOutline':
+            {
+              borderColor: 'neutral.80',
+            },
+        },
+      },
+    },
+    MuiFormControl: {
```

Comment:
This file seems large and may be difficult to maintain, can we explore breaking it into modules with named exports and group everything on here? e.g. `form.ts, table.ts, typography.ts, etc`.

Reply:
In the scope of the current task, I just copied/pasted this theme file from the theme package as it is.

But I agree, and I plan to do that in future work. We already have separate tasks for redesigning components. I will break this file implementing these tasks.